UPDATING.md

```diff
@@ -29,6 +29,8 @@ assists people when migrating to a new version.
 
 ### Breaking Changes
 
+- [26343](https://github.com/apache/superset/issues/26343): Removes the deprecated `ENABLE_EXPLORE_DRAG_AND_DROP` feature flag.
```

Comment:
Is there merit in stating that the feature flag has been removed because drag-and-drop is now the only mechanism? Someone without full context could misinterpret the current comment that this feature has been removed.

Reply:
Added `The feature is now permanently enabled.`